apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -128,28 +146,19 @@ Cloud Spanner is fully ACID compliant.
 
 Retries should be implemented at the transaction level (as opposed to individual call level)
 because there is a chance in high stress situations that a Cloud Spanner session (transaction)
-can be canceled due to causing a deadlock. In this case, the entire transaction should be
-retried and run on a new session.
+can be aborted due to causing a deadlock. In this case, the entire transaction should be
+retried. The methods `SpannerConnection.RunWithRetriableTransactionAsync(...)` will take care of
+automatically retrying the entire callback passed as a paremeter using a fresh transaction
+if previous attempts aborted.
 
-Use the extension method IsTransientSpannerFault() on Exception to create your detection
-strategy for the TransientFaultApplication Block.
+- The callback doesn't need to handle the transaction lifecycle.
+- The callback needs to set the transaction received on all the commands it creates.
+- The callback might be called more than once so it needs to be prepared for it.
```

Comment:
Possibly "prepared" is an unfortunate term, due to the idea of preparing statements etc. How about:

"The callback might be called more than once, so the code should be written bearing this in mind."